### PR TITLE
docs: add example and readme for using fromJSON

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Addresses #152

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,6 +245,56 @@ main();
...
+  try {
+    const auth = new GoogleAuth();
+    // load the JWT or UserRefreshClient from the keys
+    const client = auth.fromJSON(keys);
+    client.scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+    await client.authorize();
+    const url = `https://www.googleapis.com/dns/v1/projects/${keys.project_id}`;
+    const res = await client.request({url});
+    console.log(res.data);
+  } catch(e) {
```
Comment (ofrobots) [inline README.md]: nit: I think the example will be simpler if you dropped the try-catch from here and changed the `main` invocation to:

```js
main().catch(console.error);
```

Or you can drop the try-catch completely, and assume the user knows how to deal with errors.

File: README.md [hunk 2]
```diff
@@ -245,6 +245,56 @@ main();
...
+    console.error(e);
+  }
+  process.exit();
```
Comment (ofrobots) [inline README.md]: Why is this here?

File: README.md (hunk 1 above)
Comment (JustinBeckwith) [reply README.md]: Nice!  I went with the `main().catch` approach.

File: README.md (hunk 2 above)
Comment (JustinBeckwith) [reply README.md]: Not needed, got rid of it.
